### ISSUE TITLE
Angular NvD3 Multichart is not working.

### ISSUE THREAD

**Issue by clsahongraph** [krispo/angular-nvd3#421]
I want to create Multichart graph with Line and MultiBar Chart by using Angularjs Nv3. I have to create Line graph but unable to showing Multibar chart, i think i make some mistake. Here is [plunker](http://plnkr.co/edit/AEU93DKvpDnHL92prwnb?p=preview)

**Comment by AlexandreDenis** [comment]
Indeed you make a mistake. What didn't you create you bar chart the same way you did for the line chart ? The only thing which changes is that the type is 'bar' and not 'line'.

**Comment by clsahongraph** [comment]
No, I don't create bar graph. I need to create a graph with stack and line in nvD3 [just like here](http://codepen.io/animaldna/pen/jboqXV/). Another things here that i have a option to define own x axis.
Thanks for reply.

**Comment by blowsie** [comment]
@clsahongraph does this example help?
http://krispo.github.io/angular-nvd3/#/linePlusBarWithFocusChart

**Comment by j2kenton** [comment]
I had a very similar problem, and the following solution fixed it:

http://stackoverflow.com/questions/36047869/angular-nvd3-stacked-bar-chart-with-line
